scrc/dataset_creation/dataset_creator.py

```diff
@@ -709,14 +827,15 @@ def split_random(self, df, split):
         :param split:   the exact split (how much of the data goes into train, val and test respectively)
         :return:
         """
+        # TODO add secret_test
         train, val, test = dd.from_pandas(df, npartitions=1).random_split(list(split), random_state=self.seed)
 
         # get pandas dfs again
         train = train.compute(scheduler='processes')
         val = val.compute(scheduler='processes')
         test = test.compute(scheduler='processes')
 
-        return train, val, test
+        return train, val, test, []
```

Comment:
Is there a reason why the last value in the tuple is not a dataframe but a list?

Reply:
removed